src/main/java/edu/asu/plp/tool/backend/isa/events/RegWatchResponseEvent.java

```diff
@@ -2,32 +2,82 @@
 
 import javafx.beans.property.LongProperty;
 
+/**
+ * @brief Event to denote Register Watch Result. It returns a reference of the register object.
+ * 	      It offers communication between the front-end and the back-end
+ */
 public class RegWatchResponseEvent {
 
+	/**
+	 * @brief Boolean Variable to store the status of the request result(True or False)
+	 */
 	private boolean success;
+
+	/**
+	 * @brief String variable to hold the registerID
+	 */
 	private String registerID;
+
+	/**
+	 * @brief String variable to hold the registerName
+	 */
 	private String registerName;
+
+	/**
+	 * @brief LongProperty class defines a Property wrapping a long value.
+	 */
 	private LongProperty regObject;
 
+	/**
+<<<<<<< HEAD
```

Comment:
Remove this part. You only require keeping one part, when resolving the conflict.